### PR TITLE
06_bead_optimization_plate

### PR DESCRIPTION
**Pull request by ge73xip**

### CODE REVIEW

**Comment by armingeiser** [comment]
Same comments as on the other PR

- please use png for figures (except actual vector graphics)
- objective graph

additionally:
- please run the same example with a smaller step size, e.g. 1.5 - the shape evolution is zig-zagging a lot, i think that would improve the behaviour.

**Comment by ge73xip** [comment]
> Same comments as on the other PR
> 
> * please use png for figures (except actual vector graphics)
> * objective graph
> 
> additionally:
> 
> * please run the same example with a smaller step size, e.g. 1.5 - the shape evolution is zig-zagging a lot, i think that would improve the behaviour.

I have made the changes suggested. I think some text might be needed for image table.

**Comment by ge73xip** [comment]
> Same comments as on the other PR
> 
> - please use png for figures (except actual vector graphics)
> - objective graph
> 
> additionally:
> - please run the same example with a smaller step size, e.g. 1.5 - the shape evolution is zig-zagging a lot, i think that would improve the behaviour.
   

I m assuming smaller radius meaning smaller "filter" radius. 
With smaller filter radius the depth further reduces. So I ran with 0.085 and compared with 0.075 (existing)
Left image is with 0.075 and right one is with 0.085

**Comment by armingeiser** [comment]
Ok, then keep 0.075 as it is. Thanks

**Comment by ge73xip** [comment]
Changes made:
- images with step size 0.075. more depth to the beads
- eliminated zigzagging in gif
- comparison table of manual vs optimized solution

**Comment by armingeiser** [comment]
Few last points: 

* please use this image instead of the mesh + the pixeled version (also it does not have to be larger than it is here.

bead_analysis_setup

* don't forget to add this use case in all the toplevel markdowns as a link

File: shape_optimization/06_bead_optimization_plate/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Results
+
+### Shape Evolution
+The below images shows the shape evolution of the bead pattern on a plate during the optimization iterations. In the table further below, a comparison with manual solutions shows that the bead pattern obtained from the optimized solutions perform better. 
+
+<p align="center">
+    <img src="images/beadOpt_result.gif" width="800">
+</p>
+
+|             Manual_Solution 1              |             Manual_Solution 2              |             Manual_Solution 3              |             Manual_Solution 4              |             Manual_Solution 5              |          Optimized_Solution           |
```
Comment (armingeiser) [inline shape_optimization/06_bead_optimization_plate/README.md:37]:
```suggestion
|             Manual Solution 1              |             Manual Solution 2              |             Manual Solution 3              |             Manual Solution 4              |             Manual Solution 5              |          Optimized Solution           |
```

**Comment by ge73xip** [comment]
I updated those last points along with adding links in toplevel readme files. 
For changing the structure to "use cases", shall i create a new branch once this branch is merged ?

**Comment by armingeiser** [comment]
> For changing the structure to "use cases", shall i create a new branch once this branch is merged ?

Yes, please create a new branch/PR where you only move existing files and update links - no other changes so it is easy to review.

**Pull request merged**